filer/locale/de/LC_MESSAGES/django.po

```diff
@@ -45,6 +45,11 @@ msgid_plural "All %(total_count)s selected"
 msgstr[0] "%(total_count)s ausgewählt"
 msgstr[1] "Alle %(total_count)s ausgewählt"
 
+#: admin/folderadmin.py:431
```

Comment:
Could you remove this? Translations are handled on transifex and we cannot merge translations from github